utils/model_wrappers/tests/test_config.yaml

```diff
@@ -0,0 +1,212 @@
+embedding_model: 
+    type: "cpu"
+    batch_size: 1
+    bundle: True
+    select_expert: "e5-mistral-7b-instruct"
```

Comment:
update this to sambastudio to realy test the langchain wrapper, we can use "e5-mistral-7b-instruct-8192" model